src/main/java/model/LottoPrizeConstants.java

```diff
@@ -0,0 +1,20 @@
+package model;
+
+public enum LottoPrizeConstants {
+    FIRST_PRIZE_MONEY(2_000_000_000),

```

Comment:
지금은 단순히 상수 값을 저장하는 용도로만 사용되고 있는데, 이런 식으로 enum을 사용한다면 연관된 여러개의 데이터를 함께 처리할 수 있을 것으로 보여요

```suggestion
   FIRST(6, 2_000_000_000, "6개 일치"),
    SECOND(5, 30_000_000, "5개 일치, 보너스볼 일치"),
```

보너스볼 당첨 정보도 함께 다룬다면 로직들이 더 간단해지지 않을까요?
```suggestion
   FIRST(6, 2_000_000_000, false,"6개 일치"),
    SECOND(5, 30_000_000, true, "5개 일치, 보너스볼 일치"),
```